### PR TITLE
 Completed TL translations for 3-0-migration-guide.rst - 1,258 words 

### PR DESCRIPTION
**Pull request by josteinfrancois**
Completed translations for 3-0-migration-guide.rst file. :)

### CODE REVIEW

File: tl/appendices/3-0-migration-guide.rst [hunk 1]
```diff
@@ -944,156 +946,177 @@ Helper has had the following methods removed:
...
+
+At saka sa mas malaking mga pagbabagong ito, ilang mas maliit na nakakasirang mga
+pagbabago ang nagawa rin. Ang mga pagbabagong ito ay dapat tumulong sa pag-streamline
+sa mga binuo ng HTML FormHelper at magbawas ng mga problema na nakasalubong sa 
+mga tao sa nakaraan:
+
+- Ang ``data[`` na prefix ay natanggal mula sa lahat ng nabuong mga input.
+  Ang prefix ay wala nang tunay na layunin na pinagsisilbihan.
+- Ang iba't ibang standalone na input na mga paraan katulad ng ``text()``, ``select()``
+  at iba pa ay hindi na bumubuo ng id na mga katangian.
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: at iba pa ay to at ang iba pa ay

File: tl/appendices/3-0-migration-guide.rst [hunk 2]
```diff
@@ -944,156 +946,177 @@ Helper has had the following methods removed:
...
 ----------
 
-- ``TimeHelper::__set()``, ``TimeHelper::__get()``, and  ``TimeHelper::__isset()`` were
-  removed. These were magic methods for deprecated attributes.
-- ``TimeHelper::serverOffset()`` has been removed.  It promoted incorrect time math practices.
-- ``TimeHelper::niceShort()`` has been removed.
+- Ang ``TimeHelper::__set()``, ``TimeHelper::__get()``, at  ``TimeHelper::__isset()`` ay
+  tinanggal. Ito ay ang madyik na mga paraan para sa hindi na nagagamit na mga katangian.
+- Ang ``TimeHelper::serverOffset()`` ay tinanggal. Ito ay nagtataguyod ng hindi wastong
+  time match na mga gawi.
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: time match to time na tugma

File: tl/appendices/3-0-migration-guide.rst [hunk 3]
```diff
@@ -13,7 +13,7 @@ Mga Kinakailangan
 - Ang CakePHP 3.x ay nangangailangan ng mbstring na ekstensiyon.
 - Ang CakePHP 3.x ay nangangailangan ng intl na ekstensiyon.
 
-.. babala::
+.. warning::
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst:16]: i think it fits more on babala

File: tl/appendices/3-0-migration-guide.rst [hunk 4]
```diff
@@ -1327,7 +1327,7 @@ Seguridad
 - Ang ``Security::rijndael()`` ay hindi na ginagamit at magagamit lamang kapag
   gumagamit ng mcrypt.
 
-.. babala::
+.. warning::
```
Comment (mind-blast) [inline tl/appendices/3-0-migration-guide.rst]: i think it fits more on babala

**Comment by mind-blast** [comment]
Please find time to check my recommendation. Thank you.

File: tl/appendices/3-0-migration-guide.rst (hunk 3 above)
Comment (josteinfrancois) [reply tl/appendices/3-0-migration-guide.rst:16]: `.. note::` is a directive for sphinx and should not be translated. `.. warning::` is also a directive that should not be translated.

File: tl/appendices/3-0-migration-guide.rst (hunk 1 above)
Comment (josteinfrancois) [reply tl/appendices/3-0-migration-guide.rst]: already changed

File: tl/appendices/3-0-migration-guide.rst (hunk 2 above)
Comment (josteinfrancois) [reply tl/appendices/3-0-migration-guide.rst]: changing it to tugmang oras instead. thanks

**Review by josteinfrancois — COMMENTED** [review]
Hey @robin-ho. Thank you for your suggestions. I have updated the PR. :)

**Pull request merged**